INSTALL.MD

```diff
@@ -27,19 +28,32 @@ echo "#!/bin/bash" > /home/root/xovi/start
 echo "systemctl stop xochitl; killall xochitl 2>/dev/null; screen -dm bash -c '/home/root/xovi/_start'" >> /home/root/xovi/start
 echo "QML_DISABLE_DISK_CACHE=1 LD_PRELOAD=/home/root/xovi/xovi.so xochitl" >> /home/root/xovi/_start
 chmod a+x /home/root/xovi/{_start,start}
+
```

Comment:
You should run `mount -o remount,rw /` before running these commands, as they won't be preserved on reboot otherwise.

Reply:
As far as I know, @asivery isn't looking to overwrite the root, as this is meant to be a tethered thing; people can reboot their device to reset to stock. To preserve the changes, one would have to unmount the `/etc` overlayfs then mount the root as writable.